src/client/providers/jediHelpers.ts

```diff
@@ -49,8 +49,8 @@ export function extractSignatureAndDocumentation(definition: proxy.IAutoComplete
 }
 
 export function highlightCode(docstring: string): string {
-    // Add blank line
-    docstring = EOL + EOL + docstring + EOL + EOL;
```

Comment:
Oh, I see the key point, **blank line is matched with two consecutive** `EOL` (looking at the blank line between `fun1` and `fun2`).